be/src/olap/tablet_meta.h

```diff
@@ -183,6 +183,23 @@ class TabletMeta {
 
     OLAPStatus set_partition_id(int64_t partition_id);
 
+    bool is_beta_rowset_preferred() const {
```

Comment:
why not make this funcion perferred_rwoset_type() which return perferred rowset_type.

Reply:
ok